eng/Version.Details.xml

```diff
@@ -18,6 +18,11 @@
       <Sha>86caedcb8d09e5631a8139aab1e1cb90726668f5</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.GenAPI" Version="7.0.0-beta.22473.4">
+      <Uri>https://github.com/dotnet/arcade</Uri>
+      <Sha>86caedcb8d09e5631a8139aab1e1cb90726668f5</Sha>
+      <SourceBuild RepoName="arcade" ManagedOnly="true" />
```

Comment:
This line needs to be removed.  Only one dependency can be declared SourceBuild per repo.